### PR TITLE
Updating image dependencies and migrated pipeline to Drone v1 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,35 +1,102 @@
...
+  arch: amd64
+
+steps:
+- name: test_api
+  pull: if-not-exists
+  image: docker/compose:alpine-1.25.5
+  environment:
+    DOCKER_HOST: tcp://127.0.0.1:2375
+  commands:
+  - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
```
Comment (tasharnvb) [inline .drone.yml]:
```suggestion
  - n=0; while [ "$n" -lt 60 ] && [ ! "$(docker stats --no-stream)" ]; do n=$(( n + 1 )); sleep 1; done
```
To fix the `sh: stats: unknown operand` error in the Drone build

File: .drone.yml [hunk 2]
```diff
@@ -1,35 +1,102 @@
...
+    # - build_test_clamav-mirror
+  when:
+    event:
+    - push
+
+- name: tagged_images_to_quay
+  pull: if-not-exists
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+  commands:
+  - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
```
Comment (tasharnvb) [inline .drone.yml:64]: Don't really need this since we already wait for docker to start in the `test_api` step

File: .drone.yml [hunk 3]
```diff
@@ -1,35 +1,99 @@
...
+    DOCKER_USERNAME: ukhomeofficedigital+acp_clamav
+  when:
+    event:
+    - tag
+
+- name: latest_images_to_quay
+  pull: if-not-exists
+  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+  commands:
+  - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
```
Comment (tasharnvb) [inline .drone.yml]: Same as above

**Review by tasharnvb — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (ghost) [reply .drone.yml:64]: the `test_api` step doesn't run during a tag event so i'll need to keep it here. I'll remove it in the below step

File: .drone.yml (hunk 1 above)
Comment (ghost) [reply .drone.yml]: made the change

File: .drone.yml (hunk 3 above)
Comment (ghost) [reply .drone.yml]: made the change

File: .drone.yml (hunk 2 above)
Comment (tasharnvb) [reply .drone.yml:64]: Ah right, my mistake 👍🏾

**Pull request merged**